src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -2288,14 +2283,15 @@ Json::Value NetworkOPsImp::getServerInfo (bool human, bool admin, bool counters)
         }
         if (loadFactorFeeEscalation !=
                 escalationMetrics.referenceFeeLevel &&
-                    (admin || loadFactorFeeEscalation != loadFactor))
+                (admin ||
+                loadFactorFeeEscalation != FeeLevel64{ loadFactor }))
```

Comment:
In the process of writing this comment, I realized that the straightforward cast was just plain _wrong_! It works because the base for server load and the base for fee levels happen to be the same, but (a) that's not guaranteed to be true forever, and (b) this is exactly the kind of problem that this whole PR is supposed to be solving. 

So thanks for pointing this out!